src/olympia/templates/photon-footer.html

```diff
@@ -20,24 +20,34 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
             </ul>
         </section>
 
-        <section class="Footer-firefox-links">
-            <h4 class="Footer-links-header"><a href="https://www.mozilla.org/firefox/">Firefox</a></h4>
+        <section class="Footer-browsers-links">
+            <h4 class="Footer-links-header">Browsers</a></h4>
```

Comment:
I see a closing `</a>` tag on this line, but no matching opening tag. I believe we just need to remove this closing tag.

Reply:
Thanks, rectified.